src/examples/draggable/DraggableContent.tsx

```diff
@@ -23,8 +23,8 @@ const StyledDecorator = styled.div`
 `;
 
 const Example = () => (
-  <Row justifyContent="center">
-    <Col sm={7}>
+  <Grid.Row justifyContent="center">
+    <Grid.Col sm={7}>
```

Comment:
Should the "Oranges, mandarins, limes, and the like" `<Span>` and `StyledDecorator` receive `getColor({ variable: 'foreground.subtle' })` treatment? Or are we waiting for updated `<Span hue="foreground.subtle">` here?

Reply:
`Draggable` brings along `foreground.default`, otherwise this woul definitely need the additional color support (like the other one you mentioned).